### DISCUSSION TITLE
Welcome to wechat-gptbot Discussions!

### DISCUSSION THREAD

**Discussion by iuiaoin** [iuiaoin/wechat-gptbot#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by etfromchina** [comment]
Hi Declan, can you add those functions as follows:
1. gpt sends messages separately rather than in a whole big paragraph. 
2. gpt can send messages as scheduled.
Many thanks

**Reply by iuiaoin** [reply]
Thanks for your advice @etfromchina, I'm not sure what is "send messages separately", could you describe in detail or give a sample? If you want to make gpt reply with limited words in response at one time, you can just describe it in prompt to let gpt do this.

Scheduled messages sounds reasonable and I will consider implementing it after I finish main features.

**Reply by etfromchina** [reply]
Thanks for the reply. I hope that GPT can send information in segments instead of a whole segment, which is more in line with human reading habits. for example:
'''
Me: The weather is fine today.
Me: We can hang out together, what do you think?
Me: By the way, you can bring the dog.
'''
instead of:
'''
Me: The weather is fine today. We can take the dog out for a walk, don't you think?
'''

Btw, I've send you an email in your Gmail box a couple of days ago, not sure if you got it or not.
Again, thanks Declan.

**Reply by iuiaoin** [reply]
OK, got your point, feel free to open the feature request issue for me to track this～

Oh let me check my mailbox, I think it might be filtered by the mail rules😅
